### ISSUE TITLE
LSF: Limiting number of cores per node on with LSF?

### ISSUE THREAD

**Issue by raminammour** [JuliaParallel/LSFClusterManager.jl#12]
Hello,

How do i limit the number of cores per node with the `LSFManager`?

I would normally use `-R "span[ptile=...]"` but the option seems to be ignored if passed as an option to lsf. 

I am assuming it is because of the job array launching each core separately? @bjarthur would probably know :)

Thank you!

**Comment by DilumAluthge** [comment]
@DrChainsaw Do you have any familiarity with the `-R` flag?

**Comment by DrChainsaw** [comment]
Yes, a little bit but I'm not sure if my experience with it is helpful here. Fwiw [-R](https://www.ibm.com/docs/en/spectrum-lsf/10.1.0?topic=o-r) is a bit of a swiss army knife that allows you to put requirements on the machines running the jobs. I have not used the span requirement myself, but from the [above post](https://github.com/JuliaParallel/LSFClusterManager.jl/issues/12#issuecomment-2603425676) and the manual it should put a requirement that more than `span` different machines shall be used. As the issue appears to demonstrate, all workers end up on the same machine even then span is 2.

Assuming the problem lies with this package, my prime suspect is that the resource string is somehow dropped through addprocs_lsf, for example because something is wrong with the quoting. 

Just evaluating the command it on my computer gives this:
```julia
julia> `-R "span[ptile=2]"`
`-R 'span[ptile=2]'`
```
I see the double quotes are changed to single quotes. Perhaps LSF requires double quotes here? @DilumAluthge: Do you happen to know if there is a way to keep the double quotes?

@raminammour: There should be some output from LSF, either in an email or in some logfile if you provided that which mirrors the job submission command received by the LSF scheduler. If you happen to have access to that and post it here that would be helpful.

I guess you have moved on now though so I will try the -R flag on my system when I find the time.

**Comment by DilumAluthge** [comment]
> I see the double quotes are changed to single quotes. Perhaps LSF requires double quotes here? [@DilumAluthge](https://github.com/DilumAluthge): Do you happen to know if there is a way to keep the double quotes?

Hmmm. Not that I know off the top of my head.

**Comment by raminammour** [comment]
I lost access to the machine with the LSF manager, so I won't be able to help/debug much anymore. Sorry!
